### PR TITLE
Add support for showing the Live AuctionVC inside a SansSerifVC

### PR DESCRIPTION
**Pull request by orta**
Wraps up the presentation work for going from a URL -> LiveAuctions 👍

### CODE REVIEW

File: Artsy/View_Controllers/Debug/Quicksilver/ARQuicksilverViewController.m [hunk 1]
```diff
@@ -110,14 +111,6 @@ - (void)searchBarUpPressed:(ARQuicksilverSearchBar *)searchBar
 
 - (void)searchBarReturnPressed:(ARQuicksilverSearchBar *)searchBar;
 {
-    if ([searchBar.text hasPrefix:@"/"]) {
-        [self.presentedViewController dismissViewControllerAnimated:NO completion:nil];
-        [self addObjectToRecents:searchBar.text];
-        id controller = [ARSwitchBoard.sharedInstance loadPath:searchBar.text];
-        [self.navigationController pushViewController:controller animated:YES];
-        return;
-    }
```
Comment (orta) [inline Artsy/View_Controllers/Debug/Quicksilver/ARQuicksilverViewController.m]: I''m not too sure why this existed, so I've removed it for now. Leaves all the work to exist in the same place

File: Artsy_Tests/View_Controller_Tests/App_Menu/ARTopMenuViewControllerSpec.m [hunk 2]
```diff
@@ -26,7 +26,7 @@ - (void)presentTrialWithContext:(enum ARTrialContext)context success:(void (^)(B
 @end
 
 
-@interface ARTopMenuViewController (Testing) <ARTabViewDelegate>
+@interface ARTopMenuViewController (ExposedForTesting) <ARTabViewDelegate>
```
Comment (orta) [inline Artsy_Tests/View_Controller_Tests/App_Menu/ARTopMenuViewControllerSpec.m]: I added another category that was `Testing` earlier

File: Podfile.lock [hunk 3]
```diff
@@ -375,6 +375,6 @@ SPEC CHECKSUMS:
   VCRURLConnection: accd771ebd4be11183a3e4d5a4403f180a9a235e
   XCTest+OHHTTPStubSuiteCleanUp: 4469ec8863c6bc022c5089a9b94233eb3416c5ee
 
-PODFILE CHECKSUM: 6ac59d5af12bdd16428f017ba7fcfe937c6938d8
+PODFILE CHECKSUM: b0da427bf0fe56abbbd24a9fa2190fb65d33afbc
```
Comment (orta) [inline Podfile.lock]: could have not been in the last push?

**Comment by alloy** [comment]
Where’s the top menu VC part that shows nav VCs modally?

**Comment by orta** [comment]
Is in https://github.com/artsy/eigen/pull/1435 = [here](https://github.com/artsy/eigen/pull/1435/files#diff-5e288c4f056824e33443ac15e9689833L352)

**Comment by alloy** [comment]
Ah right, cool 👍

**Pull request merged**